src/EnergyPlus/DXCoils.cc

```diff
@@ -6061,7 +6061,7 @@ namespace DXCoils {
 					FieldNum = 1;
 					TempSize = DXCoil( DXCoilNum ).RatedTotCap( Mode );
 					SizingString = DXCoilNumericFields( DXCoilNum ).PerfMode( Mode ).FieldNames( FieldNum ) + " [W]";
-					CalcVRFCoilCapModFac( 0, _, CompName, FinalZoneSizing ( CurZoneEqNum ).DesCoolCoilInTemp, _, _, _, DataTotCapCurveValue);
+					CalcVRFCoilCapModFac( 0, _, CompName, ZoneSizingRunDone ? FinalZoneSizing ( CurZoneEqNum ).DesCoolCoilInTemp : 26, _, _, _, DataTotCapCurveValue);
```

Comment:
@rongpengzhang I appreciate the shorthand here, and it's always a tough balance for me between conciseness and clarity, but in this case I'd prefer to have the `ZoneSizingRunDone ? FinalZoneSizing ( CurZoneEqNum ).DesCoolCoilInTemp : 26` portion pulled into a separate line/variable above.